docs/reference/commandline/container_restart.md

```diff
@@ -22,3 +22,33 @@ Restart one or more containers
 ```console
 $ docker restart my_container
 ```
+
+
+### <a name="signal"></a> Stop container with signal (-s, --signal)
+
+The `--signal` flag sends the system call signal to the container to exit.
+This signal can be a signal name in the format `SIG<NAME>`, for instance
+`SIGKILL`, or an unsigned number that matches a position in the kernel's
+syscall table, for instance `9`.
```

Comment:
Maybe call out the reference here?

```suggestion
syscall table, for instance `9`. See signal(7) for available signals.
```

Reply:
Ah, yes, good suggestion.

TBH, I was somewhat wondering if we should discourage uses of numbers (as looking up the number may be better handled on the environment where it's executed), but probably better to look at that separately.

Reply:
Updated; I added a link, and used "Refer to" instead of "See" (ISTR we had a linter that didn't like "See ...", but maybe I'm mistaken 🙈